docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,214 +1,179 @@
+</Tabs>
+
+## Step 3/4. Configure Windows access
+
+In order to gain access to a remote desktop, a Teleport user needs to have the
+appropriate permissions for that desktop.
+
+Create the file `windows-desktop-admins.yaml`:
 
 ```yaml
-version: v3
-teleport:
-  auth_token: /path/to/token
-  proxy_server: teleport.example.com # replace with your proxy address
-windows_desktop_service:
-  enabled: yes
-  ldap:
-    # Port must be included for the addr.
-    # LDAPS port is 636 by default,
-    # e.g. example.com:636
-    addr: "$LDAP_SERVER_ADDRESS"
-    domain: "$LDAP_DOMAIN_NAME"
-    username: '$LDAP_USERNAME'
-    # This should be the path to the certificate exported in Step 4.
-    der_ca_file: /path/to/cert
-  discovery:
-    base_dn: "*"
-auth_service:
-  enabled: no
-proxy_service:
-  enabled: no
-ssh_service:
-  enabled: no    
+kind: role
+version: v6
+metadata:
+  name: windows-desktop-admins
+spec:
+  allow:
+    windows_desktop_labels:
+      "*": "*"
+    windows_desktop_logins: ["Administrator", "alice"]
 ```
-</TabItem>
-<TabItem label="Standalone Binary">
 
-From the directory containing the Teleport binary:
+You can restrict access to specific hosts by defining values for
+`windows_desktop_labels`, and adjust the array of usernames this role has access
+to in `windows_desktop_logins`.
+
+<Admonition type="warning" title="RBAC Configuration">
+Ensure that each Teleport user is only assigned Windows logins that they should
+be allowed to access.
+</Admonition>
+
+Apply the new role to your cluster:
 
 ```code
-$ ./teleport start --config=/etc/teleport.yaml
+$ tctl create -f windows-desktop-admins.yaml
 ```
 
-Copy the join token to a file on the instance where you will run Windows Desktop
-Service, and then use the following configuration:
+(\!docs/pages/includes/add-role-to-user.mdx role="windows-desktop-admins" \!)
 
-```yaml
-version: v3
-teleport:
-  auth_token: /path/to/token
-  proxy_server: mytenant.teleport.sh # replace with your cloud tenant
-windows_desktop_service:
-  enabled: yes
-  ldap:
-    # Port must be included for the addr.
-    # LDAPS port is 636 by default,
-    # e.g. example.com:636
-    addr: "$LDAP_SERVER_ADDRESS"
-    domain: "$LDAP_DOMAIN_NAME"
-    username: '$LDAP_USERNAME'
-    # This should be the path to the certificate exported in Step 5.
-    der_ca_file: /path/to/cert
-  discovery:
-    base_dn: "*"
-auth_service:
-  enabled: no
-proxy_service:
-  enabled: no
-ssh_service:
-  enabled: no    
-```
-</TabItem>
-</Tabs>
+## Step 4/4. Connect
 
-The access wizard will detect when the new Teleport instance has joined the
-cluster, and you can then click **Next**.
+You can now connect to your Windows desktops from the Teleport Web UI:
 
-Teleport will discover available Windows desktops in the domain, and list them
-under **Desktops**. Click **Finish** then **BROWSE EXISTING RESOURCES** to see them:
+![Connecting to a Windows desktop from the Web UI](../../img/desktop-access/non-ad-connect.png)
```

Comment:
I recommend updating this screenshot, as the nav for Teleport 12 has changed, and it would be odd if the docs for a feature that is available in v12+ showed a UI from pre-v12.

Reply:
Updated